airflowHPC/dags/replex.py

```diff
@@ -290,16 +290,21 @@ def get_swaps(iteration, dhdl_store):
         data = json.load(f)
 
     swap_list = []
-    swap_pattern = list(range(NUM_SIMULATIONS))
     state_ranges = copy.deepcopy(STATE_RANGES)
 
     dhdl_files = [
-        data["iteration"][str(iteration)][i]["dhdl"] for i in range(NUM_SIMULATIONS)
+        data["iteration"][str(iteration)][i]["dhdl"]
```

Comment:
I changed these to be a more verbose list comprehension because the fact we are reading from json (which is basically a dict) means that the lists could theoretically be "out of order".